.github/workflows/build.yml

```diff
@@ -0,0 +1,77 @@
+name: Build
+
+on: [pull_request]
+
+jobs:
+
+  static-check:
+    name: Static Analysis
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/setup-go@v2
+      with:
+        go-version: 1.12
+    - uses: actions/checkout@v2
+      with:
+        path: src/github.com/aws/amazon-ecs-agent
+    - name: run static checks
+      run: |
+        export GOPATH=$GITHUB_WORKSPACE
+        export PATH=$PATH:$(go env GOPATH)/bin
+        cd $GITHUB_WORKSPACE/src/github.com/aws/amazon-ecs-agent
+        make get-deps
+        make static-check
+
+  # unit-tests:
```

Comment:
why not just remove instead commenting out?

Reply:
I'm not removing, this is the new github action file, they are just here for when/if we decide to migrate our unit tests from travis to github